ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision/BulkSync.hs

```diff
@@ -131,6 +131,7 @@ import Control.Monad.Trans.Maybe (MaybeT (MaybeT, runMaybeT))
 import Control.Monad.Writer.Strict (Writer, runWriter, MonadWriter (tell))
 import Control.Tracer (Tracer, traceWith)
 import Data.Bifunctor (first, Bifunctor (..))
+import Data.DList (DList, fromList, toList)
```

Comment:
Import `fromList`/`toList` `qualified` (e.g. to avoid confusion if this module also uses `fromList`/`toList` from `Data.Map`)